src/components/templates/newsAndEvents/newsAndEvents.html

```diff
@@ -0,0 +1,95 @@
+<div class="newsEvents">
+  <div class="container news-events">
+    <div class="news">
+      <div class="news-and-events-title">
+        <h2>News<span class="caption">お知らせ</span></h2>
+      </div>
+      <ul class="list">
+        <li>
+          <time>2018.06.20</time>
```

Comment:
```suggestion
          <time datetime="2018-06-20">2018.06.20</time>
```
time属性は決まった書式しか受け入れていないので、仕様に厳密に則るとするとこのように `datetime` 属性を設定することになります。
ただ News コンポーネントは実装済みなので一旦修正は不要です。